### PR TITLE
Load AkkaPersistenceLivenessProbeProvider settings

### PR DESCRIPTION
**Pull request by izavala**

### CODE REVIEW

File: src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// -----------------------------------------------------------------------
+// <copyright file="UnitTest1.cs" company="Petabridge, LLC">
```
Comment (Aaronontheweb) [inline src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs]: File name is wrong

File: src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                            provider = ""Akka.HealthCheck.Persistence.AkkaPersistenceLivenessProbeProvider, Akka.HealthCheck.Persistence""
+}}  }";
+
+
+        [Fact]
+        public void AkkaPersistenceLivenessProbeProviderSettingsTest_Should_Load()
+        {
+            var healthCheck = AkkaHealthCheck.For(Sys);
+            healthCheck.Settings.Misconfigured.Should().BeFalse();
+            healthCheck.Settings.LivenessProbeProvider.Should().Be(typeof(AkkaPersistenceLivenessProbeProvider));
```
Comment (Aaronontheweb) [inline src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs]: Looks good to me

File: src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+namespace Akka.HealthCheck.Persistence.Tests
+{
+    public class AkkaPersistenceLivenessProbeProviderSettingsTest : TestKit.Xunit.TestKit
+    {
+
+        public AkkaPersistenceLivenessProbeProviderSettingsTest(ITestOutputHelper helper)
+            : base(hoconString, output: helper)
+        {
+        }
+        public static string hoconString = @"
```
Comment (Aaronontheweb) [inline src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs]: If this is going to be a `public static` member, should follow PascalCase: http://wiki.c2.com/?PascalCase - need to capitalize `h` in `hoconString`.

**Comment by Aaronontheweb** [comment]
Relates to https://github.com/petabridge/akkadotnet-healthcheck/issues/13

File: src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs (hunk 3 above)
Comment (izavala) [reply src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs]: Updated string to follow PascalCase
`public static string HoconString = @"`

File: src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs [hunk 4]
```diff
@@ -1,5 +1,5 @@
 // -----------------------------------------------------------------------
-// <copyright file="AkkaPersistenceLivenessProbeProviderSettingsTestAkkaPersistenceLivenessProbeProviderSettingsTest.cs" company="Petabridge, LLC">
+// <copyright file="AkkaPersistenceLivenessProbeProviderSettingsTest.cs" company="Petabridge, LLC">
```
Comment (Aaronontheweb) [inline src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs]: LGTM

File: src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs [hunk 5]
```diff
@@ -15,10 +15,10 @@ public class AkkaPersistenceLivenessProbeProviderSettingsTest : TestKit.Xunit.Te
     {
 
         public AkkaPersistenceLivenessProbeProviderSettingsTest(ITestOutputHelper helper)
-            : base(hoconString, output: helper)
+            : base(HoconString, output: helper)
         {
         }
-        public static string hoconString = @"
+        public static string HoconString = @"
```
Comment (Aaronontheweb) [inline src/Akka.HealthCheck.Persistence.Tests/AkkaPersistenceLivenessProbeProviderSettingsTest.cs]: LGTM

**Pull request merged**